noir-projects/noir-contracts/contracts/test/returning_tuple_contract/src/main.nr

```diff
@@ -0,0 +1,47 @@
+use aztec::macros::aztec;
+
+// Tests that Aztec.nr handles returning tuples correctly.
+#[aztec]
+pub contract ReturningTuple {
+    use aztec::{
+        macros::functions::{private, view},
+        prelude::{AztecAddress, Point},
+        protocol_types::traits::{Deserialize, FromField},
+    };
+
+    #[private]
+    #[view]
+    fn fn_that_returns_1() -> pub (bool) {
+        (true)
```

Comment:
```suggestion
    fn fn_that_returns_1() -> pub (bool,) {
        (true,)
```

These are not tuples, they are just values sorrounded by parenthesis. You need to do `(x,)` to create a single element tuple.

Reply:
e.g. this works

![image](https://github.com/user-attachments/assets/c2d43167-af12-4ecf-8e31-1501e1b6c6a2)

but this doesn't

![image](https://github.com/user-attachments/assets/04a8a66b-c423-4dd6-b82b-714a533398f5)